src/style.css

```diff
@@ -0,0 +1,5 @@
+* {
+    margin: 0;
+    padding: 0;

```

Comment:
Ficaria mais bonito modificar o margin e paddin para 20px

```suggestion
    margin: 20px;
    padding: 20px;

```

Reply:
o que voce acha de modificar para 10px?